backend/src/test/java/com/example/backend/controller/CarControllerTest.java

```diff
@@ -153,4 +153,36 @@ void addCarItem() {
 
         assertEquals(expected,actual);
     }
+
+    @Test
+    void getCarItemByID() {
+        //given
+        CarItem testCar1 = CarItem.builder()
+                .id("1")
+                .title("Testla")
+                .description("schnell")
+                .owner("owner1")
+                .involved(new ArrayList<>(Arrays.asList("involved1", "involved2")))
+                .spending("")
+                .capacity(3)
+                .trailer(false)
+                .startLocation("Garage")
```

Comment:
I have learned the last week, that unnecessary setting properties only makes the test longer.